### DISCUSSION TITLE
Pro plan: What happens after one year?

### DISCUSSION THREAD

**Discussion by jovial-snyder** [nuttyartist/notes#677] [Q&A] [answered]
https://www.get-notes.com/pricing

The `Free Pro updates for a year` part confuses me a bit. If I buy it now I will get all updates, including "Pro updates" for a year. But what happens after one year? Does the Kanban board still work? Will I always be able to download the app on new devices and all the pro features will still work just not the pro features that released after the plan expires? But even if it does, I would no longer receive any updates or bug fixes for the pro features I already paid for, correct?

**Comment by nuttyartist** [comment, marked as answer]
That is exactly correct. Sorry about the confusion. ~You might want to wait a little bit since I have plans to make the entirety of this project completely free of charge as I'm moving my efforts into Plume (https://www.get-plume.com/).~

EDIT: This plan (making the binary release free of charge is dropped for now, but might be resumed in the future).
